collectors/plugins.d/plugins_d.c

```diff
@@ -217,6 +217,11 @@ void *pluginsd_main(void *ptr)
 
     // disable some plugins by default
     config_get_boolean(CONFIG_SECTION_PLUGINS, "slabinfo", CONFIG_BOOLEAN_NO);
+    // it crashes (both threads) on Alpine after we made it multi-threaded
+    // works with "--device /dev/ipmi0", but this is not default
+    // see https://github.com/netdata/netdata/pull/15564 for details
+    if (getenv("NETDATA_LISTENER_PORT"))
```

Comment:
This env var is [set only in Docker](https://github.com/netdata/netdata/blob/33b98a003fdbf2dc3a379a7a8188f35c6bb68b44/packaging/docker/Dockerfile#L121-L122)